src/tests/test_targets.py

```diff
@@ -11,3 +13,14 @@ def test_target_deploy_nodes(self, cluster):
     @JunitTestSuite()
     def test_target_install_with_deploy_nodes(self, prepared_cluster):
         prepared_cluster.start_install_and_wait_for_installed()
+
+    @pytest.fixture
+    def download_iso_override_nodes_count(self, prepared_controller_configuration: BaseNodeConfig):
```

Comment:
can you add a docstring to explain why we need to do this?

Reply:
Sure